airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1692,65 +1692,72 @@ components:
       properties:
         logType:
           $ref: "#/components/schemas/LogType"
-    # SCHEMA
-    SourceSchema:
-      description: describes the available schema.
+    # SCHEMA CATALOG
+    AirbyteCatalog:
+      description: describes the available schema (catalog).
       type: object
       required:
         - streams
       properties:
         streams:
           type: array
           items:
-            $ref: "#/components/schemas/SourceSchemaStream"
-    SourceSchemaStream:
+            $ref: "#/components/schemas/AirbyteStreamAndConfiguration"
+    AirbyteStreamAndConfiguration:
```

Comment:
What's the point of prepending entities with `Airbyte`? Most of the other fields don't have this prefix. What is more, I do not think that it actually brings any meaning to prepend all fields with Airbyte

Reply:
Also, @ChristopheDuong do you think its a good idea to combine different parts of this object with `And`? What if we add 1 more field here somewhere in future? It means that the current name will be obsolete.

Reply:
Stream is a reserved word in java and so that's why we tend to stick the word Airbyte in front of it. We will need to come up with a better name for this this thing. The and is not ideal, but we just don't have a better name yet.